deploy-board/deploy_board/templates/hosts/host_details.tmpl

```diff
@@ -39,7 +39,9 @@
                 <td>{{ agent.failCount }}</td>
             </tr>
             <tr>
-                <td>Error Message</td>
+                <td>Error Message
+                    <a href="{{ hostdetailsvalue }}/log/deploylog?file={{env.envName}}.log">Phobos Link for {{env.envName}}</a>
```

Comment:
please add a {% if hostdetailsvalue %} for non pinterest.
and say "Full deploy log for {{env.envName}}"

Reply:
Thanks for reminding, do you mean add {% if pinterest %} for displaying phobos  links?